zombienet.toml

```diff
@@ -0,0 +1,53 @@
+[relaychain]
+command = "polkadot"
+chain = "rococo-local"
+
+[[relaychain.nodes]]
+name = "alice"
+ws_port = 9900
+
+[[relaychain.node_groups]]
+name = "bob"
+count = 4
+
+[[parachains]]
+id = 3000
```

Comment:
we should map them to the digit in the port number so its less confusing, so we know based on the port number than 9920 is 2000